packages/wallet/api/src/marshal-contexts.js

```diff
@@ -241,17 +241,19 @@ const defaultMakePresence = iface => {
  */
 export const makeImportContext = (makePresence = defaultMakePresence) => {
   const walletObjects = {
-    /** @type {IdTable<number, PurseActions>} */
```

Comment:
ouch... why did we have to take that out? Can we make it `IdTable<number, unknown>`? or even `IdTable<number, *>`? Using `*` (aka `any`) should include a justification comment.

Reply:
Added it back with `unknown`